To-do list/Controllers/TaskDetailsViewController.swift

```diff
@@ -7,39 +7,44 @@
 
 import UIKit
 
+protocol TaskDetailsViewControllerDelegate: AnyObject {
+    func sendTaskDetails(_: TaskDetailsViewController, didCreateUpdate task: Task)
```

Comment:
rename to `func taskDetails(_ controller: TaskDetailsViewController, didCreateUpdate task: Task)`

Reply:
done, renamed

Reply:
it should be  `func taskDetails(_ controller: TaskDetailsViewController, didCreateUpdate task: Task)`